source/patterns/02-organisms/site-header/site-header.twig

```diff
@@ -0,0 +1,24 @@
+{%- import 'atoms-html' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['site-header']) }) -%}
+
+<header {{- html.attributes(attributes) }}>
+
+  {%- block body -%}
+
+    {%- set logo = logo|default({})|merge({html: html}) -%}
+
+    {%- with logo only -%}
+      {%- block logo -%}
+
+        {%- set image = image|default({})|merge({src: asset('images/logo.svg')}) -%}
+        {%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['site-header__logo']) }) -%}
+
+        {%- include 'atoms-image' -%}
```

Comment:
This pattern is meant for including an image in content, rather than any image anywhere. Not a problem at the moment as there's no styling, but there will be when https://github.com/libero/pattern-library/pull/87 is added. Probably need to have an `<img>` here.